module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -4,101 +4,102 @@
 | Library/Package                    | Piece of code        | What it's called                | What it does                                                                                                      |                                                                                 
 |------------------------------------|----------------------|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
-| Base `R`          | [`library()`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/library)                    | Library   | Loads and attaches additional packages to the R environment.                                                                                                                                                  |
-| Base `R`          | [`<-`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/assignOps.html)                    | Assignment operator   | Assigns a name to something in the R environment.                                                                                                                                                  |
-| Base `R`          | [`c()`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/c)                     | Combine           | Combines values into a vector or list.                                                                                                                                                             |
+| Base `R`          | [`library()`](https://rdrr.io/r/base/library.html)                    | Library   | Loads and attaches additional packages to the R environment.                                                                                                                                                  |
+| Base `R`          | [`<-`](https://rdrr.io/r/base/assignOps.html)                    | Assignment operator   | Assigns a name to something in the R environment.                                                                                                                                                  |
+| Base `R`          | [`c()`](https://rdrr.io/r/base/c.html)                     | Combine           | Combines values into a vector or list.                                                                                                                                                             |
 | Base `R`          | [`%in%`](http://www.datasciencemadesimple.com/in-operator-in-r/)                                  | "in" logical operator           | Checks if the given value(s) on the left side of the operator are in the vector or other R object defined on the right side of the operator. It returns a logical `TRUE` or `FALSE` statement.                                                                                                                                                              |
```

Comment:
```suggestion
| Base `R`          | [`%in%`](http://www.datasciencemadesimple.com/in-operator-in-r/)                                  | "in" logical operator           | Checks if the given value(s) on the left side of the operator are in the vector or other R object defined on the right side of the operator. It returns a logical `TRUE` or `FALSE` statement.                                                                                                                                                              |
```

Do we want to link to https://rdrr.io/r/base/match.html for uniformity? We could link to both...

Reply:
I actually went back and forth on this one. I liked the `datasciencemadesimple` more because it was just about the operator, but point taken about consistency. I think both is a good call for this one. I can change the link to rdrr, and add into description `[This](datsciencemadesimple) is also useful` or similar.